portfolio/src/main/webapp/WEB-INF/DATA-DIR/WEB-INF/index.yaml

```diff
@@ -0,0 +1,2 @@
+indexes:
```

Comment:
I'm not gonna as you to remove those .yaml files now... I guess that they'll be needed 🤷🏼‍♂️

Reply:
Yeah I thought so too 😳